lib/shared/screens/admin/shared/helpers/map-settings-ids.js

```diff
@@ -0,0 +1,3 @@
+import {map, compact} from 'lodash';
+
+export default (options) => compact(map(options, (option) => option.id));
```

Comment:
Check here https://github.com/relax/relax/blob/master/lib/shared/elements/mobile-menu/style.js#L32 and https://github.com/relax/relax/blob/master/lib/shared/elements/mobile-menu/style.js#L49

Being that options is always an array and unlocks can be an array or an object.